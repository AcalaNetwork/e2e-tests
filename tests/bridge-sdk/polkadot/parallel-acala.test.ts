import { bridgeSDKTest } from '../shared'

bridgeSDKTest({
  from: 'parallel',
  to: 'acala',
  token: 'PARA',
})

bridgeSDKTest({
  from: 'parallel',
  to: 'acala',
  token: 'ACA',
})
