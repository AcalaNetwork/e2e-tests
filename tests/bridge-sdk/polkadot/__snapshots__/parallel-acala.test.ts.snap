// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'ACA' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 98,
  "toChain": "(rounded 12)",
}
`;

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'ACA' > fee 1`] = `"(rounded 0.006)"`;

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'ACA' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 100,
  "toChain": 10,
}
`;

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'PARA' > after 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": "(rounded 998)",
  "toChain": "(rounded 1.99)",
}
`;

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'PARA' > fee 1`] = `"(rounded 0.006)"`;

exports[`'parallel' to 'acala' using bridgeSDK cross-chain 'PARA' > initial 1`] = `
{
  "address": "5FA9nQDVg267DEd8m1ZypXLBnvN7SFxYwV7ndqSYGiN9TTpu",
  "decimals": 12,
  "fromChain": 1000,
  "toChain": 0,
}
`;
